app/portainer/components/forms/git-form/git-form-auto-update-fieldset/git-form-auto-update-fieldset.html

```diff
@@ -1,4 +1,10 @@
-<ng-form name="autoUpdateForm">
+<ng-form name="autoUpdateForm form-group">
+  <div class="small vertical-center">
+    <pr-icon icon="'info'" mode="'primary'"></pr-icon>
+    <span class="text-muted">
+      When enabled, at each polling interval or webhook invocation, if the git repo differs from what was stored locally on last git pull, the changes are deployed.</span
+    >
+  </div>
   <div class="form-group">
     <div class="col-sm-12">
       <por-switch-field
```

Comment:
This switch looks quite close to the info text above it.  And not centered properly with "Automatic Updates" text

![image](https://user-images.githubusercontent.com/6227712/205801777-04920836-b58b-4ce3-94e4-14fc4b9dc0a0.png)

Reply:
Nice spot - updated with the bottom margin - the switch uses a global component, so I wont update it in this ticket